src/ApolloHooksMutation.re

```diff
@@ -140,7 +146,7 @@ let useMutation:
                (
                  switch (
                    Js.Nullable.toOption(jsResult##data),
-                   Js.Nullable.toOption(jsResult##error),
+                   Js.Nullable.toOption(jsResult##errors),
                  ) {
                  | (_, Some(error)) => (Error(error): result('data))
```

Comment:
@mbirkegaard I think we should add a guard to this pattern branch `when Js.Array.length(error) > 0`.

Reply:
Yep.